src/logic/function/iterate.lean

```diff
@@ -149,4 +153,7 @@ theorem right_inverse.iterate {g : α → α} (hg : right_inverse g f) (n : ℕ)
   right_inverse (g^[n]) (f^[n]) :=
 hg.iterate n
 
+lemma comm {m n : ℕ} : f^[n]^[m] = (f^[m]^[n] : α → α) :=
```

Comment:
This creates `function.comm`. Did you mean `iterate_comm`? Otherwise LGTM.

Reply:
Yes, indeed!  Thanks for noticing!